### PR TITLE
fix(Tooltip): align arrow correctly when tooltip is close to viewport's edge

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1688)

### CODE REVIEW

File: src/slider.css.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export const container = style([
     }),
     {
         touchAction: 'none',
+        userSelect: 'none',
```
Comment (marcoskolodny) [inline src/slider.css.ts:20]: Unrelated change: prevent user select when long pressing the slider thumb

File: src/tooltip.tsx [hunk 2]
```diff
@@ -486,6 +486,10 @@ const Tooltip: React.FC<Props> = ({
                                   exit: TOOLTIP_EXIT_TRANSITION_DURATION_IN_MS,
                               }
                     }
+                    onExited={() => {
```
Comment (marcoskolodny) [inline src/tooltip.tsx:489]: We need to reset tooltip computed positions after it's closed

File: src/__screenshot_tests__/__image_snapshots__/tooltip-screenshot-test-tsx-tooltip-arrow-appears-properly-in-vivo-new-skin-when-target-is-close-to-viewport-edges-1-snap.png
Comment (marcoskolodny) [inline src/__screenshot_tests__/__image_snapshots__/tooltip-screenshot-test-tsx-tooltip-arrow-appears-properly-in-vivo-new-skin-when-target-is-close-to-viewport-edges-1-snap.png:1]: Removed margin from target in tooltip's story to show the case in which the arrow is too close to the viewport's edge

File: src/__screenshot_tests__/tooltip-screenshot-test.tsx [hunk 3]
```diff
@@ -107,3 +108,16 @@ test('Tooltip - active tooltip is closed if another one is opened', async () =>
     const thirdTooltip = await page.screenshot();
     expect(thirdTooltip).toMatchImageSnapshot();
 });
+
+test('Tooltip - arrow appears properly in Vivo_New skin when target is close to viewport edges', async () => {
```
Comment (atabel) [inline src/__screenshot_tests__/tooltip-screenshot-test.tsx:112]: Please, add a comment explaining why the "in Vivo_New skin" part is relevant (I guess it's because of the border radius)

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]
